### PR TITLE
Batch delete in delete_entities tool

### PR DESCRIPTION
**Pull request by bfcrampton**

### CODE REVIEW

File: util/delete_entities.py [hunk 1]
```diff
@@ -26,24 +26,21 @@ def parse_args():
...
-        entity_splits = entity.split('\t')
-        fiss_args = FissArgs(True, args.workspace_namespace,
-                             args.workspace_name, entity_splits[0],
-                             entity_splits[1])
-        fiss.entity_delete(fiss_args)
+    # associated participant_set/sample/sample_set.
+    for entity_type in ['participant', 'sample', 'participant_set', 'sample_set', 'pair']:
+        if entity_type in entities_by_type:
+            entities = entities_by_type[entity_type]
+            fapi.delete_entity_type(args.workspace_namespace, args.workspace_name, entity_type, entities)
```
Comment (melissachang) [inline util/delete_entities.py]: Can you call [fapi._check_response_code](https://github.com/broadinstitute/fiss/blob/master/firecloud/fiss.py#L374)

And print a message either way (delete succeeded, or delete failed)

File: util/delete_entities.py [hunk 2]
```diff
@@ -26,24 +26,21 @@ def parse_args():
...
-    # participant_set, participant.
-    for entity in reversed(entities):
-        print('Deleting ' + entity)
-        entity_splits = entity.split('\t')
-        fiss_args = FissArgs(True, args.workspace_namespace,
-                             args.workspace_name, entity_splits[0],
-                             entity_splits[1])
-        fiss.entity_delete(fiss_args)
+    # associated participant_set/sample/sample_set.
+    for entity_type in ['participant', 'sample', 'participant_set', 'sample_set', 'pair']:
```
Comment (melissachang) [inline util/delete_entities.py]: What about other entity types, like cohort?

Comment (bfcrampton) [reply util/delete_entities.py]: Added.

File: util/delete_entities.py (hunk 1 above)
Comment (bfcrampton) [reply util/delete_entities.py]: Done.

File: util/delete_entities.py [hunk 3]
```diff
@@ -26,25 +26,37 @@ def parse_args():
...
+            entities = entities_by_type[entity_type]
+            fapi.delete_entity_type(args.workspace_namespace, 
+                                    args.workspace_name, entity_type, entities)
+            fapi._check_response_code(resp, 200)
+            print 'Succesfully deleted entities of type: %s' % entity_type
+            del entities_by_type[entity_type]
+
+    # Delete the remaining entities where order does not matter.
+    for entity_type, entities in entities_by_type.iteritems():
+        fapi.delete_entity_type(args.workspace_namespace,
```
Comment (melissachang) [inline util/delete_entities.py]: For clarity, put line 44 in a constant. Add here: `if entity_type not in firecloud_types:`

Comment (bfcrampton) [reply util/delete_entities.py]: Done.

**Pull request merged**